modules/monitoring-optimizing-alerting-rules-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: CONCEPT
+[id="Optimizing-alerting-rules-for-core-platform-monitoring{context}"]
+= Optimizing alerting rules for core platform monitoring
+
+Consider these key recommendations before creating or modifying alerting rules for core platform monitoring.
+By following these guidelines, you can optimize your alerting system, which helps you to build an efficient and effective monitoring environment tailored to your organization's specific needs
+
+* *Minimize the number of new rules*.
+Create only rules that are essential to your specific requirements.
+By minimizing the number of rules, you create a more manageable and focused alerting system in your monitoring environment.
+
+* *Focus on symptoms rather than causes*.
+Create rules that notify users of symptoms instead of underlying causes.
+This approach ensures that users are promptly notified of a relevant symptom so that they can investigate the root cause after an alert has triggered.
+This tactic also significantly reduces the overall number of rules you need to create.
```

Comment:
I have a drastic suggestion you might want to explore if you haven't done so already:
These two bullet points seem to be relevant only to [this section](https://62211--docspreview.netlify.app/openshift-enterprise/latest/monitoring/managing-alerts.html#creating-new-alerting-rules_managing-alerts).I would have moved these two points there as either two bullet points in a TIP before `.Prerequisites` or two additional intro paragraphs.

Reply:
That's a thought-provoking sugested change that would really change the whole approach. Thanks! , but a) I want to stay consistent with the structure of the parallel topic for user-defined projects at https://docs.openshift.com/container-platform/4.13/monitoring/managing-alerts.html#Optimizing-alerting-for-user-defined-projects_managing-alerts, and b) I think it's a better for users to see all of the tips aggregated in a single place at the beginning of whole section versus spreading them over different topics -- sort of a "before you begin" approach to the whole process. 

So I'm going to leave this as-is.